packages/financial-templates-lib/src/helpers/GasEstimator.ts

```diff
@@ -256,12 +260,12 @@ export class GasEstimator {
   }
 
   private _extractFastGasPrice(json: { [key: string]: any }, url: string): LondonGasData | LegacyGasData {
-    if (url.includes("etherchain.org")) {
+    if (url.includes("etherscan.io")) {
       const etherchainResponse = json as EtherchainResponse;
-      if (etherchainResponse.recommendedBaseFee === undefined) throw new Error(`Bad etherchain response ${json}`);
+      if (etherchainResponse.result.suggestBaseFee === undefined) throw new Error(`Bad etherchain response ${json}`);
       return {
-        maxFeePerGas: etherchainResponse.recommendedBaseFee,
-        maxPriorityFeePerGas: etherchainResponse.fastest,
+        maxFeePerGas: etherchainResponse.result.suggestBaseFee,
+        maxPriorityFeePerGas: etherchainResponse.result.FastGasPrice,
```

Comment:
Few interesting links

- https://docs.alchemy.com/docs/maxpriorityfeepergas-vs-maxfeepergas

-  https://ethereum.org/en/developers/docs/gas/#:~:text=Since%20each%20Ethereum%20transaction%20requires,of%20transaction%20success%20or%20failure.&text=Gas%20fees%20are%20paid%20in,currency%2C%20ether%20(ETH).